content/en/v3/admin/platforms/minikube/_index.md

```diff
@@ -24,6 +24,8 @@ This guide will walk you though how to setup Jenkins X on your laptop using [min
 
 * [Install minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
 
+* [Install virtualbox](https://www.virtualbox.org/wiki/Downloads)
```

Comment:
But doesn't the command say to use `--driver=docker`?

Reply:
The actual command says `--vm=true`.

![Screen Shot 2021-10-01 at 2 42 19 PM](https://user-images.githubusercontent.com/6811830/135689323-42b2f5b1-a5db-46d4-8c8d-9ebd4408ea76.png)

The `NOTE` where `--driver=docker` is recommended is targeted to Windows 10 Home users. 

Since I was following this on a mac, I simply ignored the note and `--vm=true` failed because I did not have virtualbox installed and it was not listed in the Prerequisites list.